compliance/google/long_stopped_instances/README.md

```diff
@@ -18,6 +18,13 @@ The policy leverages the Google API to check all instances that have been stoppe
 
 - credential_viewer
 
+### Actions (REQUIRED)
```

Comment:
remove (REQUIRED)

I see this is in the README_GUIDELINE and I will removed.

Reply:
I have removed. Please review